FreeRTOS/FreeRTOS-Kernel
# Add more comment for core affinity in prvSelectHighestPriorityTask

Description
-----------
Add more comments for core affinity in `prvSelectHighestPriorityTask` to explain the logic. Forum user feeds back questions about the core select logic for core affinity feature. This PR adapts some the discussion in the comment to help reader understand the logic in this function.

Test Steps
-----------


Checklist:
----------



Related Issue
-----------
[Link to forum questions](https://forums.freertos.org/t/smp-question-about-the-function-prvselecthighestprioritytask/17926/7)


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
